### PR TITLE
Client dashboard

### PR DESCRIPTION
**Pull request by heron2014**
## Ready to review

Related to #453 

@nelsonic please have a look at this PR, thank you.

Let us know before merging to add the rejected property with empty array to all the candidates in db.

### CODE REVIEW

**Comment by nelsonic** [comment]
@heron2014 has *current* `master` been merged into the `client-dashboard` branch ?

**Comment by heron2014** [comment]
@nelsonic yes, unfortunately it is merged with master. Huge PR, we need to learn to make smaller ones!

File: assets/css/client/main.css [hunk 1]
```diff
@@ -1,95 +1,522 @@
+@font-face {
+  font-family: 'Myriad-Pro';
+  src: url(Myriad-Pro_31655.ttf) format('truetype');
```
Comment (nelsonic) [inline assets/css/client/main.css]: Where did we get this Font from? (_Typekit_?)

File: assets/css/client/main.css [hunk 2]
```diff
@@ -1,95 +1,522 @@
...
 }
 
 .logo img {
-  display: block;
   margin-left: auto;
   margin-right: auto;
-  width: 30em;
-  margin-top: 20px;
+  width: 8em;
+  margin-bottom: 80px;
```
Comment (nelsonic) [inline assets/css/client/main.css]: try not to mix `em` and `px` ...

File: assets/css/client/main.css [hunk 3]
```diff
@@ -1,95 +1,522 @@
...
+.cl-container:nth-child(3) {
+  margin-top: 100px;
+}
+
+.cl-job {
+  margin-top: 30px;
+}
+
+.cl-job-info {
+  display: -webkit-flex;
```
Comment (nelsonic) [inline assets/css/client/main.css]: could we _condense_ all the `display: flex;` instances into a single definition to reduce duplication?

File: assets/js/client_dashboard.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
+function findAncestor (el, cls) {
+  while ((el = el.parentElement) && !el.classList.contains(cls));
```
Comment (nelsonic) [inline assets/js/client_dashboard.js]: this code _deserves_ a _clarifying comments_...

File: lib/api.js [hunk 5]
```diff
@@ -77,7 +77,10 @@ exports.register = function (server, options, next) {
                       request.payload.statusHistory = response.hits.hits[0]._source.statusHistory;
 
                       //keep notesList
-                      request.payload.notesList - response.hits.hits[0]._source.notesList;
+                      request.payload.notesList = response.hits.hits[0]._source.notesList;
```
Comment (nelsonic) [inline lib/api.js]: is there _always_ a hit?

**Comment by nelsonic** [comment]
@heron2014 / @SimonLab code looks good. I would prefer to see a few more ✅  (_fewer failing CI builds_) and a focus on keeping code [_DRY_](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
But the functionality is there and I'm happy to merge. 👍

**Pull request merged**